Lib/ufo2ft/__init__.py

```diff
@@ -70,17 +73,19 @@ def call_preprocessor(ufo_or_ufos, *, preProcessorClass, **kwargs):
     return preProcessor.process()
 
 
+@timer("compile a basic TTF")
 def call_outline_compiler(ufo, glyphSet, *, outlineCompilerClass, **kwargs):
     kwargs = prune_unknown_kwargs(kwargs, outlineCompilerClass)
     outlineCompiler = outlineCompilerClass(ufo, glyphSet=glyphSet, **kwargs)
     return outlineCompiler.compile()
 
 
 def call_postprocessor(otf, ufo, glyphSet, *, postProcessorClass, **kwargs):
-    if postProcessorClass is not None:
-        postProcessor = postProcessorClass(otf, ufo, glyphSet=glyphSet)
-        kwargs = prune_unknown_kwargs(kwargs, postProcessor.process)
-        otf = postProcessor.process(**kwargs)
+    with timer("postprocess TTF"):
```

Comment:
this could be a `@timer` decorator as the with statement basically wraps all the lines, no?

Reply:
The odd thing is that when I tried that I always got a time of 0.00s before everything was executed.

Reply:
that's strange. If we can repro we can try fix

Reply:
it would still be nice to use the decorator here for consistency, or explain in a comment why not